aws-kendra-datasource/src/main/java/software/amazon/kendra/datasource/Translator.java

```diff
@@ -120,23 +120,21 @@ static DeleteDataSourceRequest translateToDeleteRequest(final ResourceModel mode
    * @return awsRequest the aws service request to modify a resource
    */
   static UpdateDataSourceRequest translateToUpdateRequest(final ResourceModel model) {
-    String description = model.getDescription() == null ? "" : model.getDescription();
-    String name = model.getName() == null ? "" : model.getName();
-    String roleArn = model.getRoleArn() == null ? "" : model.getRoleArn();
-    String schedule = model.getSchedule() == null ? "" : model.getSchedule();
```

Comment:
Setting RoleArn and schedule as Blank, caused Invalid RoleArn and Invalid schedule exceptions.